firebase-installations/src/main/java/com/google/firebase/installations/local/PersistedInstallationEntry.java

```diff
@@ -90,6 +93,9 @@ public abstract Builder setRegistrationStatus(
     @NonNull
     public abstract Builder setTokenCreationEpochInSecs(long value);
 
+    @NonNull
+    public abstract Builder setFISException(@Nullable String value);
```

Comment:
why can this be @Nullable?

Reply:
When PersistedInstallationEntry stores a successfully registered entry.